.github/workflows/unit-tests.yml

```diff
@@ -125,13 +125,30 @@ jobs:
             os: ubuntu-24.04-arm
             DISTRIB: conda
             LOCK_FILE: build_tools/github/pymin_conda_forge_arm_linux-aarch64_conda.lock
+
+          # Linux environment to test the latest available dependencies.
+          # It runs tests requiring lightgbm, pandas and PyAMG.
+          - name: Linux pylatest_pip_openblas_pandas
+            os: ubuntu-24.04
+            DISTRIB: 'conda-pip-latest'
```

Comment:
I looked at `install.sh`, I think this is an historical quirk [^1], this can be replaced by this now:
```suggestion
            DISTRIB: 'conda'
```

The fact that packages are installed with pip in already part of the build name `pylatest_pip_...` so I don't think there is much point of keeping it in the `DISTRIB`.

[^1]: probably we were handling things differently in the past in `install.sh` but now the fact that we are installing the packages with `pip` is part of the lock-file.